dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/bnd.bnd

```diff
@@ -47,11 +48,19 @@ tested.features:\
 
 -buildpath: \
 	com.ibm.ws.componenttest;version=latest,\
+	com.ibm.websphere.javaee.servlet.4.0;version=latest,\
+	com.ibm.websphere.javaee.jaxrs.2.1;version=latest,\
+	com.ibm.websphere.javaee.cdi.1.2;version=latest,\
 	io.openliberty.microprofile.telemetry.internal_fat.common,\
 	io.openliberty.io.opentelemetry;version=latest,\
+	com.ibm.websphere.org.eclipse.microprofile.rest.client.1.3;version=latest,\
+	com.ibm.websphere.javaee.annotation.1.3;version=latest,\
+	com.ibm.websphere.javaee.concurrent.1.0;version=latest,\
```

Comment:
You don't need any of these additional dependencies because they're already included further down the list.